gem/README.md

```diff
@@ -0,0 +1,68 @@
+# EarlGrey
+
+[![Gem Version](https://badge.fury.io/rb/earlgrey.svg)](https://rubygems.org/gems/earlgrey)
+
+Automatically installs EarlGrey. Supports carthage, cocoapods, and Swift.
+
+```
+$ earlgrey help install
+Usage:
+  earlgrey install -t, --target=TARGET
+
+Options:
+  -p, [--project=PROJECT]            # Project
+  -t, --target=TARGET                # EarlGrey
+  -s, [--scheme=SCHEME]              # EarlGrey.xcscheme
+      [--swift], [--no-swift]
+                                     # Default: true
+      [--carthage], [--no-carthage]
+                                     # Default: true
+
+Installs EarlGrey into an Xcode unit test target
+```
+
+## Usage
+
+Create new Target `iOS Unit Testing Bundle` then add a new scheme
+`Product → Scheme → Manage Schemes` and mark it as shared.
+See [EarlGrey docs for screenshots](https://github.com/google/EarlGrey/blob/master/docs/install-and-run.md)
```

Comment:
Unless you meant for users to read the install guide from the begining, this link should take us directly to the appropriate screenshots: https://github.com/google/EarlGrey/blob/master/docs/install-and-run.md#step-1-set-up-a-test-target